src/pydagmc/dagnav.py

```diff
@@ -584,6 +589,34 @@ def reverse_volume(self) -> Optional[Volume]:
     def reverse_volume(self, volume: Volume):
         self.senses = [self.forward_volume, volume]
 
+    @property
+    def _boundary_group(self) -> Optional[Group]:
+        for group in self.groups:
+            if "boundary:" in group.name:
+                return group
+        return None
+
+    @property
+    def boundary(self) -> Optional[str]:
+        """Name of the boundary assigned to this surface."""
+        group = self._boundary_group
+        if group is not None:
+            return group.name[9:]
```

Comment:
Seeing as we're now requiring Python 3.9

```suggestion
            return group.removeprefix('boundary:')
```

Reply:
Nice, didn't know about that method!